README.md

```diff
@@ -67,8 +67,11 @@ To start the local dev-environment all the components can be started at once by
 ./start_all.sh
 ```
 
-Note: Before first run, you need to run `cd maker-frontend; yarn install; cd../taker-frontend; yarn install` command to ensure that all dependencies get
-installed.
+Note: Before first run, you need to install all dependencies
+
+- `cd maker-frontend; yarn install; cd../taker-frontend; yarn install`
+- `brew install protobuf` // see https://docs.rs/prost-build/latest/prost_build/#sourcing-protoc for more details.
+- `brew install cmake` // adapt accordingly to your os.
```

Comment:
this is macOC specific; I doubt that most of the github audience is.

maybe provide macOS and linux variants? (or just mention that `protobuf` and `cmake` need to be installed?)

Reply:
since I am not developing on linux, I am not sure of the linux (or windows for that matter) variants - so I am only going to mention it and add the brew command as reference.

Reply:
> interesting.. what OS do you think the github audience uses the most? :) I feel like there majority runs on macOS, but I might be living in a bubble 😅.

then you certainly live in a bubble 😂 I would have said "Linux", but latest SO dev survey shows that most devs use Windows - source:  https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-operating-system
not that surprising, considering github is owned by Microsoft and WSL got better over the years.